tests/backup/backup_kubevirt_test.go

```diff
@@ -3438,9 +3438,17 @@ var _ = Describe("{KubevirtVMMigrationTest}", Label(TestCaseLabelsMap[KubevirtVM
 					dash.VerifyFatal(err, nil, fmt.Sprintf("Verfiying VM [%s] is up and running after rebooting the node [%s]", vm.Name, nodeName))
 					log.Infof("Validating all px-backup pods are ready after reboot")
 					err = ValidateAllPodsInPxBackupNamespace()
-					dash.VerifyFatal(err, nil, "px-backup pods verification successful")
-					//Adding a sleep of 2 minute, Sometimes, it takes time for the pod to become ready.
-					time.Sleep(120 * time.Second)
+					dash.VerifyFatal(err, nil, "verifing px-backups pods are in running state after reboot")
+					t := func() (interface{}, bool, error) {
+						PxBackupVersion, err := GetPxBackupVersionString()
+						if err != nil {
+							return nil, false, fmt.Errorf("failed to get px-backup version string")
```

Comment:
Please print the error as well so that we know that is the error and take actions in case it is a fatal issue

Reply:
Done